EnglishFilter/sections/antiadblock.txt

```diff
@@ -1317,6 +1317,8 @@ falixnodes.net#@#.adsbygoogle
 !#endif
 ! https://github.com/AdguardTeam/AdguardFilters/issues/73214
 psychpoint.com##.adblockmodal
+! https://github.com/AdguardTeam/AdguardFilters/issues/86215
+@@||hwupgrade.it^$generichide
```

Comment:
Find the section with `#%#//scriptlet("abort-current-inline-script", "atob", "googlefc")` and add a domain to those rules.